engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -239,8 +244,14 @@ class WorkflowManagerActor(params: WorkflowManagerActorParams)
       timers.cancel(RetrieveNewWorkflowsKey)
       sender() ! akka.Done
       goto(RunningAndNotStartingNewWorkflows)
+    case Event(SubWorkflowStart(actorRef), data) =>
+      // Watch for this subworkflow to expire to remove it from the Set of running subworkflows.
+      context.watch(actorRef)
+      stay using data.copy(subWorkflows = data.subWorkflows + actorRef)
+    case Event(Terminated(actorRef), data) =>
+      stay using data.copy(subWorkflows = data.subWorkflows - actorRef)
```

Comment:
I don't _think_ it matters that we go here even if the terminated actor is not a subworkflow actor (as removing it will do nothing), but it could deserve a comment explicitly stating it, as the goal here is really to count dead subworkflow actors but we're doing it for any dead actor that we're watching.